interface/src/scripting/MenuScriptingInterface.h

```diff
@@ -60,22 +57,23 @@ private slots:
 
 public slots:
     /**jsdoc
-     * Add a new top-level menu.
+     * Adds a new top-level menu.
      * @function Menu.addMenu
-     * @param {string} menuName - Name that will be displayed for the menu. Nested menus can be described using the ">" symbol.
+     * @param {string} menuName - Name that will be displayed for the menu. Nested menus can be specified using the 
+     *     <code>"&gt;"</code> character.
```

Comment:
```suggestion
     *     <code>"&gt;"</code> symbol.
```
I added this for consistency with the same definition in the MenuItemProperties object (menuName)

Reply:
Changed the MenuItemProperties JSDoc to have "character" instead of "symbol".